external_services/cache.py

```diff
@@ -12,6 +12,7 @@ def __init__(self, course_instance):
         super().__init__(course_instance)
 
     def _generate_data(self, instance, data=None): # pylint: disable=arguments-differ
+        empty_string = ''
```

Comment:
Is the line `empty_string = ''` required?

Reply:
It's the remnant from the first commit. I will remove that.